.github/workflows/tests.yml

```diff
@@ -616,7 +618,7 @@ jobs:
 
     - name: Run Elasticsearch
       run: |
-        docker run -d -p 9200:9200 -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xms128m -Xmx256m" elasticsearch:7.9.2
+        docker run -d -p 9200:9200 -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xms128m -Xmx256m" -e "ELASTIC_CLIENT_APIVERSIONING=1" -e "xpack.security.enabled=false" elasticsearch:8.6.0
```

Comment:
Can you read these env values somehow from services/ elasticsearch/ env?

Reply:
I could set them in the env, but those would not be automatically propagated to `docker run`, leading to more verbosity - for example, say we only have one var to set, ELASTIC_CLIENT_APIVERSIONING=1, the step would look like this:
```
    - name: Run Elasticsearch
      env: 
         ELASTIC_CLIENT_APIVERSIONING: 1
      run: |
        docker run -d -p 9200:9200 -e ELASTIC_CLIENT_APIVERSIONING elasticsearch:8.6.0
```

Not sure it's worth it, also considering we have a bunch of steps like that in this job and that this job is going to be eventually rewritten using services.

Reply:
Ok cool - didn't know